zipkin-autoconfigure/collector-kafka10/README.md

```diff
@@ -0,0 +1,130 @@
+# Kafka 0.10+ Collector
+
+This collector is implemented as a Kafka consumer supporting Kafka brokers running version 0.10.0.0 
+or later. To activate this collector, reference the module jar when running the Zipkin server 
+and configure one or more bootstrap brokers via the `KAFKA_BOOTSTRAP_SERVERS` environment
+variable or `zipkin.collector.kafka.bootstrap-servers` property.
+
+## Quick start
+
+The minimum version of Java required to run Zipkin server and the Kafka 0.10+ collector is JRE 8.
```

Comment:
JRE7 is the minimum requirement for using kafka-clients, including version 0.10. I copied the JRE8 requirement from the zipkin-server readme.

There's no reason collector-kafka10 can't be JRE 7 compatible. I'll check the code to see if there's anything Java 8 specific.